meta-lhg-wpe/recipes-wpe/wpewebkit/wpewebkit_0.1.bbappend

```diff
@@ -1,16 +1,28 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 # override WPEWebKit till ClearKey PR is merged
-BASE_URI = "git://github.com/psivasubramanian/WebKitForWayland;protocol=git;branch=patch-8"
-SRC_URI_append = "file://0001-incompatitable-types-fix.patch"
-SRCREV = "233a70860b59c47a7c6bdda237412f987be0c4f2"
+# clearkey validated from 'patch-8' branch
+# with 'pr_working' branch, no proper url available to validate clearkey
```

Comment:
The comment seems a bit contradictory. What URL did we use to validate ClearKey with patch-8 branch?

Reply:
We had validated ClearKey through Youtube URL from the same SRC_URI with 'patch-8' branch. Later some gstreamer qtdemux patches had been integrated which are from RDK Central that is mandatory to have to parse Multi DRM enabled media source. So with these changes, the validated ClearKey Youtube URL is not played from any of the mentioned SRC_URI branch. There is no more support from ClearKey Youtube media source and also we couldn't find a proper URL for ClearKey validation in WPE browser.